locale/es_ES.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -106,8 +106,8 @@ msgstr "Agregar"
 
 #: index.php:980
 msgid "Sorry! Screensharing for your device is currently not supported."
-msgstr ""
-"Lo lamentamos! Tu dispositivo no está soportado para compartir pantalla."
+msgstr "Lo lamentamos! Tu dispositivo no está soportado para compartir pantalla."
+
```

Comment:
That will be re-formatted automatically to the old format by the `gettext` software which splits "long" texts.